tests/C/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`C.sol 1`] = `
+pragma solidity >0.6.4 <0.7.0;
+
+contract C {
+    uint constant X = 32**22 + 8;
+    string constant TEXT = "abc";
+    bytes32 constant MY_HASH = keccak256("abc");
+    uint immutable decimals;
+    uint immutable maxBalance;
+    address immutable owner = msg.sender;
+
+    constructor(uint _decimals, address _reference) public {
+        decimals = _decimals;
+        // Assignments to immutables can even access the environment.
+        maxBalance = _reference.balance;
+    }
+
+    function isBalanceTooHigh(address _other) public view returns (bool) {
+        return _other.balance > maxBalance;
+    }
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+pragma solidity >0.6.4 <0.7.0;
+
+
+contract C {
+    uint256 constant X = 32**22 + 8;
+    string constant TEXT = "abc";
+    bytes32 constant MY_HASH = keccak256("abc");
+    uint256 decimals;
+    uint256 maxBalance;
```

Comment:
This is solving only the parser error, `prettier-solidity` is not printing the immutable key

Reply:
you are right, let me mark this PR still as WIP, any suggestion?

Reply:
As far as I can see, the parser isn't giving us any information right now.
there is an issue open https://github.com/solidity-parser/parser/issues/17
For the moment we wait.
 @fvictorio any comments?

Reply:
Ugh, yeah, we'll need to add support for that in the parser. I'll work on that asap.

Reply:
See https://github.com/solidity-parser/parser/pull/18